### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by DuduHerolo**

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -3,10 +3,9 @@ const app = express()
...
 const es6Promise = require('es6-promise')
-const apiRouter = require('./routers')
-const package = require('./package')
-const config = require('./config')
-const { force_https } = require('./middlewares')
+const apiRouter = require('../routers')
+const appPackage = require('../package')
+const { force_https } = require('../middlewares')
 
 es6Promise.polyfill()
```
Comment (maxpolski) [inline app/index.js]: on which version of node it will run? i mean the application

File: bin/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const config = require('../config')
+const app = require('../app')
+
+const server = require('http').Server(app)
+
+server.listen(config.port, function(){
+  console.log('Server is listening on port %s', 3000)
```
Comment (maxpolski) [inline bin/index.js]: i think it should be like this `console.log('Server is listening on port %s', config.port)`

File: controllers/email.js [hunk 3]
```diff
@@ -23,7 +23,13 @@ exports.getEmail = async (req, res) => {
     const data = await api.fetchRelevantData(hotelId, reservationUuid)
     const output = await renderer.render(type, data)
 
-    res.send(output)
+    const { email_sender, email_subject } = data.deal.texts[0]
```
Comment (maxpolski) [inline controllers/email.js]: if you suppose that destructuring is working, then you definitely shouldn't polyfill the Promise

File: helpers/api.js [hunk 4]
```diff
@@ -20,8 +20,7 @@ const fetchReservation = (resId) => new Promise((resolve, reject) => {
       if (res.ok) {
         return res.json().then(resolve)
       } else {
-        const { status, statusText } = res
-        reject(errors.resolveStatusToError(status, statusText))
+        reject(errors.resolveStatusToError(res.status, res.statusText))
       }
     })
     .catch(err => {
```
Comment (maxpolski) [inline helpers/api.js]: why don't you use async/await here?

File: app/index.js (hunk 1 above)
Comment (DuduHerolo) [reply app/index.js]: v10.15.3

**Pull request merged**